src/tardev-snapshotter/tarindex/src/lib.rs

```diff
@@ -447,14 +455,20 @@ pub fn append_index(data: &mut (impl io::Read + io::Write + io::Seek)) -> io::Re
             group: e.group.into(),
             lmtime: (e.mtime as u32).into(),
             size: e.size.into(),
-            offset: e.offset.into(),
+            offset: inode_offset,
```

Comment:
nit: call `.into()` here instead of up there - more readable IMO

Reply:
good point. updated.